src/test/scala/org/hammerlab/guacamole/likelihood/LikelihoodSuite.scala

```diff
@@ -18,32 +18,74 @@
 
 package org.hammerlab.guacamole.likelihood
 
-import org.hammerlab.guacamole.likelihood.LikelihoodUtil._
+import org.bdgenomics.adam.util.PhredUtils
 import org.hammerlab.guacamole.pileup.Pileup
 import org.hammerlab.guacamole.reads.MappedRead
-import org.hammerlab.guacamole.util.{GuacFunSuite, TestUtil}
-import org.hammerlab.guacamole.variants.Genotype
+import org.hammerlab.guacamole.util.{Bases, GuacFunSuite, TestUtil}
+import org.hammerlab.guacamole.variants.{Allele, Genotype}
 import org.scalatest.prop.TableDrivenPropertyChecks
 
 class LikelihoodSuite extends GuacFunSuite with TableDrivenPropertyChecks {
 
+  implicit lazy val reference = TestUtil.makeReference(sc, Seq(("chr1", 1, "C")))
```

Comment:
what does the implicit do here? I would rather avoid it if possible for clarity to non scala experts like me, unless it is doing something really useful (in which case a comment would help)